packages/menu/src/MenuItem.tsx

```diff
@@ -262,7 +262,7 @@ MenuItem.propTypes = {
   href: PropTypes.string,
   onClick: PropTypes.func,
   className: PropTypes.string,
-  description: PropTypes.string,
+  description: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```

Comment:
I'm 80% sure that PropTypes.node includes string, so you might be able to omit it.

Reply:
Good catch, I am pretty sure about that too.